core/new-gui/src/app/dashboard/component/feature-container/user-dictionary-section/ngbd-modal-resource-add/ngbd-modal-resource-add.component.ts

```diff
@@ -30,6 +32,13 @@ export class NgbdModalResourceAddComponent {
   public separator: string = '';
```

Comment:
In the Ng2FileUploader example, they include additional directives for the uploader. We should add the directives described here https://github.com/valor-software/ng2-file-upload/blob/master/demo/components/file-upload/simple-demo.ts. In addition, `styleUrls` should only contain `'./ngbd-modal-resource-add.component.scss'`

Reply:
![image](https://user-images.githubusercontent.com/39659542/56380898-3dd16f80-61c8-11e9-9d30-2299d9743083.png)
![image](https://user-images.githubusercontent.com/39659542/56380917-4a55c800-61c8-11e9-81ed-ab5740344f75.png)
They didn't use it in another demo and guide. There is no such FILE_UPLOAD_DIRECTIVES any more and it doesn't affect the function.

Reply:
I didn't touch styleUrls. I'll talk to isaac about this